plugins/auto-sizes/load.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
I question the use of "load.php" as the main plugin file once we've moved these from modules. I think a better filename would be  to use the common slug convention `plugins/auto-sizes/auto-sizes.php` or `plugins/auto-sizes/plugin.php`. Not a technical blocker, but is something that came up in the plugin review when it was first submitted. In the SVN repo, it's currently [using the former](https://plugins.trac.wordpress.org/browser/auto-sizes/trunk), I just hadn't updated the PR yet.

Reply:
Got it. Adjust in https://github.com/WordPress/performance/pull/972/commits/87f3a04c351ef37f9c3b411176a39f6399f48cb1